common-content/en/module/how-our-curriculum-works/classplanner/index.md

```diff
@@ -0,0 +1,29 @@
++++
+title="Classplanner"
+emoji="📅"
+time=10
+[objectives]
+    1="Bookmark the Classplanner"
+    2="Schedule your classes"
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
+
++++
+
+> Bookmark the [Classplanner](https://classplanner.codeyourfuture.io/)
+
+We use Classplanner to manage Saturday rotas per region, so we can all see who is volunteering and when. This is currently just for volunteers, but we are working on expanding it to include learners.
+
+### 🤹🏽 Course Leaders
+
+Course Leaders, [register your course](https://classplanner.codeyourfuture.io/London/newcourse/)s and [classes](https://classplanner.codeyourfuture.io/London/newclass/) in Class Planner and add links to the curriculum and class agenda. Share these newly-created volunteering opportunities with the CYF volunteer community, usually via Slack or the volunteer weekly newsletter.
+
+It's important to **schedule your course in advance**, so volunteers can commit and add the classes to their own calendars. Do this before the start of the course, so volunteers can plan their availability. Help your team help you!
+
+### 🙋🏽 Volunteers
+
+Volunteers, view classes scheduled in your region (choose /cities), and sign up to those which suit your expertise and availability. You can commit to a half-day or full day on your chosen dates.
```

Comment:
```suggestion
Volunteers, [view classes scheduled in your region](https://classplanner.codeyourfuture.io/cities/), and sign up to those which suit your expertise and availability. You can commit to a half-day or full day on your chosen dates.
```

Reply:
Oh I didn't do that because if they go straight there they get a 404 and then they come to Slack asking why it's broken (it's because they're not logged in). It's a bad flow people struggle with. But if they go to the home page they will get a login prompt and then they can go to cities.